src/platform/site-wide/sass/modules/_m-footer-brand-consolidation.scss

```diff
@@ -97,6 +97,7 @@
   > .va-footer-links {
     > li {
       margin-bottom: 0.5em;
+      list-style: none;
```

Comment:
fixes regression where footer list items were getting a list-style

Reply:
Looks like this is being addressed here, https://github.com/department-of-veterans-affairs/vets-website/pull/8804, as well.

Reply:
ahh good catch @ncksllvn no conflicts though, let me pull master back in